packages/adapter-static/index.js

```diff
@@ -26,11 +26,23 @@ export default function (options) {
 
 				if (dynamic_routes.length > 0) {
 					const prefix = path.relative('.', builder.config.kit.files.routes);
+					const config_option =
```

Comment:
Looking at this again, I think the best way to do this is "either dynamic routes are in the unprerendered pages or the entries option is not the default, then show this message. And if there are dynamic routes then add that sentence in the paranthesis). So basically what I had before but with `||` instead of `&&`